docs/_documentation/using_per_event_authorization.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Event-Based Authorization
+position: 13
+---
+
+## Event-Based Authorization using EventAuthField
+
+Nakadi provides an option to specify `event_auth_field` in an event type definition, which can be used to differentiate
```

Comment:
Let's be more straighforward. Something like `Nakadi provides per-event filtering, allowing event type owners to specify which users should be able to read an event`.

Then you can say that this is achieved through the field `event_auth_field`.